grpc_check/README.md

```diff
@@ -0,0 +1,63 @@
+# Agent Check: grpc_check
+
+## Overview
+
+This check monitors endpoints implementing [gRPC Health Checking Protocol][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+To install the grpc_check check on your host:
+
```

Comment:
One final note- customers can install the integration directly without building ie: https://github.com/DataDog/integrations-extras/blob/master/aqua/README.md#installation